CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # **Changelog**
 
+#### 3.0.0 (2022-04-06)
```

Comment:
Can you add also SDKCF-4826. It was about setting up sonarcloud and sonarqube and ensuring that quality gate passes (by adding more unit tests). I'm not sure how to put that nicely in once sentence, I'm bad ad this. Tanomu 🙏